src/pages/index.js

```diff
@@ -168,24 +172,16 @@ function closeModal(modal) {
 
 function handleEditFormSubmit(evt) {
   evt.preventDefault();
+  profileName.textContent = editModalNameInput.value;
+  profileDescription.textContent = editModalDescriptionInput.value;
+  closeModal(editModal);
+
   const data = {
     name: editModalNameInput.value,
     about: editModalDescriptionInput.value,
   };
 
-  setButtonText(editSubmitBtn, true);
-
-  api
-    .editUserInfo(data)
-    .then((updatedUser) => {
-      profileName.textContent = updatedUser.name;
-      profileDescription.textContent = updatedUser.about;
-      closeModal(editModal);
-    })
-    .catch(console.error)
-    .finally(() => {
-      setButtonText(editSubmitBtn, false);
-    });
+  renderLoading(cardSubmitBtn, true);
```

Comment:
Please, return the correct submit handler for the `profile form`. It should have `.then`, `.catch` and `.finally`